src/main/java/edu/hawaii/its/api/wrapper/AddMembersCommand.java

```diff
@@ -61,6 +62,21 @@ private AddMembersCommand addUid(String uid) {
         return this;
     }
 
+    public AddMembersCommand addGroupPathOwner(String groupPath) {
+        WsSubjectLookup wsSubjectLookup = new WsSubjectLookup();
+        wsSubjectLookup.setSubjectSourceId("g:gsa");
```

Comment:
Maybe a comment on line 67 would be helpful for others to understand why we need `g:gsa`.